electron/renderer/src/lib/__tests__/localStorage.spec.ts

```diff
@@ -17,17 +17,11 @@
  *
  */
 
-// eslint-disable-next-line jest/no-mocks-import
-import {LocalStorageMock} from '../__mocks__/localStorage';
 import {loadState, saveState} from '../localStorage';
 
 describe('localStorage', () => {
-  beforeEach(() => {
-    global.localStorage = new LocalStorageMock();
-  });
-
```

Comment:
do we have local storage now? 
In this case, can we remove the mock altogether?